modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -372,98 +364,265 @@ class RuleList extends Component {
+                                                </>
+                                            )}
+                                        </>
+                                    ))}
+
+                                    <br />
+
+                                    {rule.actions.map(action => {
+                                        if(!isLast(action)){
+                                            return (<>
+                                                    , <br />
+                                                    <b> {Liferay.Language.get('and')} </b>
+                                            </>);
+                                        }
+                                        return null;
+                                    })}
+
+                                </p>
+                            </div>
+
+                            <div className="list-group-item-field">
+                                <div className="card-col-field">
+                                    {
+                                        rule.invalidRule && (
+                                        <>
+                                            {Liferay.Language.get('due-to-missing-fields')}
+                                            {Liferay.Language.get('broken-rule')}
+
+                                            <div className="invalid-rule" title={titleMessage}>
+                                                <ClayLabel spritemap={spritemap} displayType="danger">
+                                                    {labelMessage}
+                                                </ClayLabel>
+                                            </div>
+                                        </>
+                                        )
+                                    }
+
+                                    <div className="dropdown dropdown-action">
+                                        <div className="ddm-rule-list-settings" data-card-id={ruleIndex}>
+                                            <ClayActionsDropdown 
+                                                items={rule.rulesCardOptions}
+                                                expanded={dropdownExpandedIndex === ruleIndex} 
+                                                itemClicked={onRuleCardClicked}
+                                                spritemap={spritemap}
+                                            />
+
+                                            <button className="component-action cursor-pointer dropdown-toggle" onClick={onDropdownClicked} data-rule-index={ruleIndex}>
+                                                <ClayIcon spritemap={spritemap} symbol="ellipsis-v" />
+                                            </button>
+                                        </div>
+                                    </div>
+                                </div>
```

Comment:
Initially you can check if [`ClayDropDownWithItems` serves you](https://clayui.com/docs/components/drop-down.html#dropdownwithitems), if not, you can customize using DropDown low-level :)

Reply:
ok